content/practices/outcome-oriented-roadmap/index.md

```diff
@@ -0,0 +1,225 @@
+- Prioritize based on business & user value
+- Establish and maintain stakeholder and cross-team alignment, collaboration and visibility
+- Create a backlog
+
+when:
+- You’ve started development but don’t have a roadmap
+- During development, you need a way to align your backlog with your goals and metrics
+- Product sponsors and stakeholders ask for more visibility into the product direction and your progre
+
+what:
+- Whiteboard
+- Post-it notes
+- Sharpies
+- Painter’s tape
+
+# If this practice or workshop has a Miro template: remote: true
+remote: false
+miro_template_url: "" 
+
+---
+## How to Use this Method
+Using outcome-oriented roadmaps, rather than organizing our roadmaps around fixed features and dates, helps us to visibly communicate what the value is of what we want to achieve while maintaining flexibility in how we implement.
+
+Outcome-oriented roadmaps are typically created during or shortly after Inception to help frame the MVP through prioritizing multiple problems/opportunities for immediate focus. However, they are living documents that change over time.
+
+This session is usually better to do once you have a product **mission**, **vision** and **strategy** already, or know the business goals and metrics your product must contribute to.
+
+### Sample Agenda & Prompts
+1. Explain the difference between an outcome and an output:  (5 minutes)
+
+   An ***Output*** **is a thing** the product does — like a feature.
+
+   An ***Outcome*** **is a measurable change in behavior** that drives business results. Outcomes describe the ‘why’ of a set of features and the value they are intended to deliver.
+
+   {{% callout %}}
+   Tip/Example: For example, for a musical instrument retailer, an outcome might be "a customer is able to see our different instruments". But this outcome can be achieved through several outputs, like sending mail order catalogs, or creating a web page they can browse, or creating an augmented reality product where a musician can see what the instrument might look like on them. Any one of these outputs could help to achieve the outcome, but one might be more successful than the other.
+   {{% /callout %}}
+
+   We don’t want to get too focused on a potentially unsuccessful output too early, so we should focus on being clear about the outcome (the "why") during roadmapping.
+
+   Your stakeholders may be more used to features-over-time roadmaps, which focus on specific features (without context of why they are needed) and fixed delivery dates, and may not have the flexibility to release the most valuable things first. In contrast, an outcome-oriented roadmap helps us to be clear about the value of what we build, and allows us to release the most valuable things first and less valuable things incrementally after that.
+
+1. Understand the Vision (5 minutes) 
+
+   If you have a vision statement for your product, write it down and put it on your board. Check to see if everyone understands it.
```

Comment:
Add final sentence: "Summarize the strategy for achieving this vision to ensure collective understanding." 

This sets the stage for recognizing that the brainstormed outcomes should be aligned with the overall strategic path towards the vision.

Reply:
Updated.

Reply:
Feel free to comment on the update.